src/Modules/OrchardCore.Commerce/Controllers/PaymentController.cs

```diff
@@ -174,24 +189,8 @@ public async Task<IActionResult> Success(string orderId)
         return View(order);
     }
 
-    [Route("success/{orderId}")]
-    [HttpPost]
-    [ValidateAntiForgeryToken]
-    public async Task<IActionResult> SuccessPost(string orderId)
+    private async Task FinalModificationAsync(ContentItem order)
```

Comment:
This method contains 
```csharp
        order.Alter<OrderPart>(part => part.Status =
            new TextField { ContentItem = order, Text = OrderStatuses.Ordered.HtmlClassify() });
```
but it's only called by `PaymentConfirmationMiddleware` if the order status is already "ordered". So this alter is not necessary.